go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/mongodb/terraform-provider-mongodbatlas
 
 go 1.14
-
+replace go.mongodb.org/atlas => ../go-client-mongodb-atlas
```

Comment:
Probably shouldn't leave, looks like for testing?

Reply:
Yes, so I can use the recent changes of client mongodb, since it's merged I will update from master mongodb client